js/src/api/rpc/parity/parity.js

```diff
@@ -95,6 +95,11 @@ export default class Parity {
       .execute('parity_dappsList');
   }
 
+  dappsRefresh () {
```

Comment:
Ah! Thanks for the review! I'll reply when I wake up in a few hours. The JS code just ensures an instant refresh.. otherwise you have to reload the page (<super>+R), and I don't think we should require that. It would be confusing to the end user.

So for instance, if you link a new dapp... how would JS know unless it constantly checked if there was a new one... [When pub-sub is ready, I can change it] but this is incredibly inefficient since the number of times a developer is going to add a new dapp is like... once a month at most?